### DISCUSSION TITLE
Simple Admin and RM521F-GL ?

### DISCUSSION THREAD

**Discussion by buggz2k** [iamromulan/cellular-modem-wiki#9] [General - Quectel]
Anyone gotten Simple Admin to work on a Quectel RM521F-GL?
My experience, seems, it tries, but not working?

**Comment by buggz2k** [comment]
Ok, FINALLY, got this going.
I have some version of MAIN working.
It has working band locking, so, I am happy.
Thanks!

**Comment by iamromulan** [comment]
Apologies I didn't see this until now. The development branch is nearly ready to push to main 😁
